app/src/main/java/tttTDD/Java/Board.java

```diff
@@ -0,0 +1,44 @@
+package tttTDD.Java;
+
+import org.checkerframework.common.value.qual.IntRange;
+
+import java.lang.reflect.Array;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+public class Board {
+    private final String[] board;
+
+    public Board() {
+        this.board = new String[] {"1", "2", "3", "4", "5", "6", "7", "8", "9"};
+    }
+
+    public Board(int boardSize) {
```

Comment:
As we discussed when pairing, I like the idea of being able to modify the array and the size of the board. I guess this wasn't an explicit requirement, but it is an example of being able to "extend" your code without having to change the class itself.